library/core/src/future/mod.rs

```diff
@@ -62,7 +62,7 @@ unsafe impl Sync for ResumeTy {}
 /// better error messages (`impl Future` rather than `GenFuture<[closure.....]>`).
 // This is `const` to avoid extra errors after we recover from `const async fn`
 // FIXME(swatinem): This fn can be removed when bumping the bootstrap compiler
```

Comment:
@Swatinem can the entire function be removed now?

Reply:
Removed the unknown lang items for now, IMO it's fine to merge this if it builds and cleanup the functions (if needed) later.

Reply:
Yes, the idea was that this whole function can be removed now, as it is now unused.

Reply:
Happy to see a separate PR with more expansive cleanup, but I think minimal work as part of this PR is best.